src/hooks/Rule/useTopology.ts

```diff
@@ -1,40 +1,13 @@
 import { onMounted, ref, onUnmounted, Ref } from 'vue'
 import { getRules, getBridgeList } from '@/api/ruleengine'
 import G6, { Graph, ModelConfig, IGroup } from '@antv/g6'
```

Comment:
I haven't used G6, but I was wondering if it would be possible to help bring the Graph part into a separate file to distinguish the logic of the Graph Register from the logic of the Rule?

Reply:
Maybe we just need to pass some Rule **Topology logic** or **Data parmas** into a function to generate a complete Topology Graph, but just my thoughts.

Reply:
Would it also be helpful to split the use of Bridge or Rule?